src/lib/share-addresses.ts

```diff
@@ -0,0 +1,7 @@
+import { WebRTC } from '@multiformats/multiaddr-matcher'
+import type { Multiaddr } from '@multiformats/multiaddr'
+
+export const getWebRTCAddrs = (addrs?: Multiaddr[]): Multiaddr[] => {
+  // TODO: Get only WebRTC addrs dialable from other browsers, e.g. WebRTC Direct, WebRTC Secure WebSockets, and WebRTC WebTransport (currently not included in Helia transports)
+  return addrs?.filter((addr: Multiaddr) => WebRTC.exactMatch(addr)) ?? []
```

Comment:
are you talking about:

```suggestion
  return addrs?.filter((addr: Multiaddr) => WebRTC.exactMatch(addr) || (Circuit.exactMatch(addr) && (WebRTCDirect.matches(addr) || WebTransport.matches(addr) || WebSockets.matches(addr) || WebSocketsSecure.matches(addr)))) ?? []
```

Reply:
You suggested change would include circuit relay multiaddrs without `webrtc`

For example, when running this PR, I get two very similar multiaddrs:
```
"/dns4/213-202-223-221.k51qzi5uqu5dm8atc646dp7gahck7cjiqn14w7osacekah21jmdy1917pf24i5.libp2p.direct/tcp/4001/tls/ws/p2p/12D3KooWS9XdnaVcYKK7QWY55UAz9YcYQfkpyqgB2sYBYZN8GQ3v/p2p-circuit/p2p/12D3KooWNCAakqex88aYV7gn4Uno3mHe1LLAmH4hf9tqxN6ghguj"
"/dns4/213-202-223-221.k51qzi5uqu5dm8atc646dp7gahck7cjiqn14w7osacekah21jmdy1917pf24i5.libp2p.direct/tcp/4001/tls/ws/p2p/12D3KooWS9XdnaVcYKK7QWY55UAz9YcYQfkpyqgB2sYBYZN8GQ3v/p2p-circuit/webrtc/p2p/12D3KooWNCAakqex88aYV7gn4Uno3mHe1LLAmH4hf9tqxN6ghguj"
```

The only difference between them is that one has `webrtc`, which would pass the `WebRTC.exactMatch`. The other one would pass `(Circuit.exactMatch(addr) && (WebRTCDirect.matches(addr) || WebTransport.matches(addr) || WebSockets.matches(addr) || WebSocketsSecure.matches(addr)))` but that is undesired, because this is not going to trigger the webrtc direct connection. 

In summary, we only want `WebRTC.exactMatch` because those will attempt a direct p2p connection.

Reply:
Fixed this in https://github.com/ipfs-shipyard/ipfs-share-files/pull/206/commits/01cf103d267ca0c36590ced1149930786c2521a3

This ensures only browser friendly circuit relay webrtc multiaddrs are embedded